docker/package/model.py

```diff
@@ -303,11 +303,11 @@ def fetch_sources(self, out_dir, binaries_dir=None):
 
         os.chdir("..")
 
-    def gen_control_file(self, deps, ubuntu_version, out):
-        str_build_deps = ", ".join(deps)
-        str_additional_native_deps = ", ".join(
-            self.__get_os_specific_native_deps("ubuntu")
+    def gen_control_file(self, build_deps, run_deps, ubuntu_version, out):
+        str_run_deps = ", ".join(
+            run_deps + self.__get_os_specific_native_deps("ubuntu")
         )
+        str_build_deps = ", ".join(build_deps)
```

Comment:
It looks to me like something went wrong with the deps handling, because I noticed that installing `tezos-baking` now requires more packages to be installed (even tho it doesn't seem like they are actually needed).

Reply:
What you see installed is packages' listed runtime deps, which for whatever reason weren't ever installed on ubuntu (but were on fedora)